src/alertsmanagement/azext_alertsmanagement/_help.py

```diff
@@ -0,0 +1,67 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=too-many-lines
+# pylint: disable=line-too-long
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['alertsmanagement action-rule'] = """
+    type: group
+    short-summary: Commands to manage alertsmanagement action rule.
+"""
+
+helps['alertsmanagement action-rule create'] = """
```

Comment:
highly suggest provide 3 examples to cover 3 scenarios listed here https://docs.microsoft.com/en-us/azure/azure-monitor/platform/alerts-action-rules#example-scenarios as speclet highlighted as breadth coverage

Reply:
Done